include/envoy/stats/histogram.h

```diff
@@ -11,6 +11,23 @@
 namespace Envoy {
 namespace Stats {
 
+using SupportedBuckets = const std::vector<double>;
+
+class HistogramSettings {
+public:
+  virtual ~HistogramSettings() = default;
+
+  /**
+   * For formats like Prometheus where the entire histogram is published (but not
+   * like statsd where each value to include in the histogram is emitted separately),
+   * get the limits for each histogram bucket.
+   * @return The buckets for the histogram. Each value is an upper bound of a bucket.
+   */
+  virtual SupportedBuckets& buckets(absl::string_view stat_name) const PURE;
```

Comment:
`SupportedBuckets` is defined as const, but it should probably be renamed to `ConstSupportedBuckets`, because I agree it's not clear when reading uses of it.